spec/models/user_spec.cr

```diff
@@ -128,4 +128,26 @@ describe User do
       expect(User.find_by_login("bad-login")).to be_nil
     end
   end
+
+  describe "#last_hour_announcements" do
+    before do
+      Announcement.clear
+      User.clear
+    end
+
+    let(:user) { user(login: "john2").tap &.save }
+
+    it "returns the number of announcements made by the user during the last hour" do
+      an = announcement(user: user).tap &.save
+      an.created_at = Time.new(2016)
```

Comment:
```
Time.new(2016) => 0001-01-01 00:00:00
```

but ok!

Reply:
That's weird, I thought it would be ```2016-01-01 00:00:00```!

Changed to ```Time.new(2016, 2, 15, 10, 20, 30)``` anyway.